CHANGELOG.md

```diff
@@ -13,7 +13,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixes
 
--
+- Fixes an [issue](https://issues.redhat.com/browse/OCPBUGS-3420) where
+  `ScanSettingBindings` created without a `settingRef` didn't use a proper
+  default value. `ScanSettingBindings` without a `settingRef` will now default
```

Comment:
```suggestion
  default value. `ScanSettingBindings` without a `settingRef` will now revert
```

Reply:
Nice catch. I only removed the words `default` and `revert` since `revert` can carry a different meaning of reverting a patch.

Hopefully the result is concise yet still understandable.